code/src/Components/EmailQuestion.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react'
+
+export const EmailQuestion = (props) => {
+    const {step, emailInput, onEmailInputChange, onStepBackChange, onStepChange} = props
+    const progress = (step/8)*100
+    return(
+        <div className="container">
+
+            <label htmlFor="emailInput" className="labelStyle">What is your email?</label>
+            <input 
+            id="emailInput"
+            type="email" 
+            value={emailInput} 
+            placeholder="name@email.com"
+            onChange={onEmailInputChange}/>
+            
+            <div className="buttons">
+                <button className="button"
+                    disabled={emailInput===''}
+                    onClick={onStepChange}
+                    >Next <span className="arrow right"></span></button>
+                <button className="button"
```

Comment:
I like that you have added disabled on the buttons if the input is empty, but also adding disable to the email so that you only can add text with @ would be good